modules/accounts/calendar.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+ 
+  config = cfg.accounts.calendar;
+
+#  calendarOpts = { name, config, ... }: {
+#    options = {
+#      name = mkOption {
+#        type = types.str;
+#        readOnly = true;
+#        description = ''
+#          Unique identifier of the calendar. This is set to the
+#          attribute name of the calendar configuration.
+#        '';
+#      };
+#    };
+#
+#    config = mkMerge [ { name = name; } ];
+#  };
+
+in
+
+{
+  options.accounts.calendar = {
+    basePath = mkOption {
+      type = types.str;
+      default = "${config.home.homeDirectory}/.calendars/";
```

Comment:
XDG_DATA_HOME would be a best default IMHO.

Reply:
An alternative is to simply leave out the default value and force the user to make an conscious decision.